src/file_saver.rs

```diff
@@ -1,10 +1,10 @@
-use std::fs;
-use std::fs::File;
-use std::io::Result;
+use std::fs::{self, File};
+use std::io::{Result, Write};
 use std::path::Path;
+use writer::untabber::Untabber;
 
 pub fn save_to_file<P, F>(path: P, make_backup: bool, closure: &mut F) where
```

Comment:
Forgot `mut` in function defininuon `pub fn save_to_file<P, F>(path: P, make_backup: bool, mut closure: F) where`
not really no reason